kazoo/protocol/connection.py

```diff
@@ -363,6 +365,18 @@ def _write(self, msg, timeout):
                     raise ConnectionDropped("socket connection broken")
                 sent += bytes_sent
 
+    def _find_persistent_watchers(self, path):
```

Comment:
Do we have to distinguish between persistent and persistent recursive watches here? Given a persistent (non-recursive) watch for `/a` and another one for `/a/b` it looks like an event for `/a/b` would currently also trigger the callback for the watch for `/a`.